src/wallet/sqlite.cpp

```diff
@@ -192,7 +192,29 @@ bool SQLiteDatabase::Rewrite(const char* skip)
 
 bool SQLiteDatabase::Backup(const std::string& dest) const
 {
-    return false;
+    sqlite3* db_copy;
+    int res = sqlite3_open(dest.c_str(), &db_copy);
+    if (res != SQLITE_OK) {
+        sqlite3_close(db_copy);
+        return false;
+    }
+    sqlite3_backup* backup = sqlite3_backup_init(db_copy, "main", m_db, "main");
+    if (!backup) {
+        LogPrintf("SQLiteDatabase::Backup: Unable to begin backup: %s\n", sqlite3_errmsg(m_db));
```

Comment:
8eee65c46d3b897fca9e07b092f3714e3bcdb79e
```suggestion
        LogPrintf("%s: Unable to begin backup: %s\n", __func__, sqlite3_errmsg(m_db));
```

Reply:
Done